doc/developer/documentationAraInstall.md

```diff
@@ -0,0 +1,67 @@
+
+   ## ARA Documentation
+
+ ###### Requirements
+ To install ARA you need :
+
+       - at least Java 8
+       - Intellij Idea (back development)
+       - Visual Studio Code ( front development )
+       - Docker
+       - git
+
+Verify docker is installed on your IDE `$ docker --version`
+Verify java is installed on your IDE `$ java --version`
+
+ ###### Get ARA source code
+
+`$ mkdir 'yourFolder'`
+`$ cd 'yourFolder'`
+`$ git clone https://github.com/Decathlon/ara.git`
+
+ ###### BDD installation
+
+Start your docker and run this command line : 
+
+`$ docker run -e MYSQL_ROOT_PASSWORD=yourDatabasePassword -p 3306:3306 decathlon/ara-db`
+
+__NB : Note your DataBasePassword, it will serve for later__
+
+  ###### Set up front code
+
+After Downloaded Visual Studio Code, verify that you have plugins below :
+- npm plugin
+- vetur plugin
+
+Import this folder in Visual Studio Code : 
+\ara\client
+
+In Visual Studio Code Terminal : enter command below :
+`$ npm install`
+`$ npm install dependancies`
+`$ npm run-script dev`
+
+  ###### Set up back code
+
+Go to Intellij and import ara project previously recovered under Github
+- Make `$ clean install `on each pom.xml folder except client folder
+
+- In Edit Configuration part :
+
+**Main Application** : com.decathlon.ara.AraApplication
+**VM Options **: -Dspring.profiles.active=dev -- DspringMake .configclean install  onaeachource.password folder=**yourDatabasePassword**

```

Comment:
I Think you have a bad Copy/Paste here 😄 
The VM options are : 
`-Dspring.profiles.active=dev -Dspring.config.location=classpath:/ -Dspring.datasource.password=**yourDatabasePassword**`

Reply:
done